entries/b/bilibili.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Bilibili": {
+    "domain": "bilibili.com",
+    "contact": {
+      "facebook": "bilibilidouga",
```

Comment:
As the CircleCI build says, the Facebook handle `bilibilidouga` is either private or doesn't exist. Please remove it.
```suggestion
```